resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1424 @@
+            "Is patiently listening to someone talk about things they are unquestionably wrong about and dying inside",
+            "Looks on as the camp buzzes around {PRONOUN/m_c/object}",
+            "Is basking in the gentle sunlight around camp",
+            "Is meditating upon a sun-warmed rock",
+            "Is sitting quietly on the far side of camp",
+            "Is humming a soothing tune to {PRONOUN/m_c/self} as {PRONOUN/m_c/subject} {VERB/m_c/work/works}",
+            "Is out on a peaceful stroll",
+            "Is remembering simpler days with fondness",
+            "Is offering gentle advice to a frustrated Clanmate",
+            "Is lining {PRONOUN/m_c/poss} nest with lavender",
+            "Smells sweet, like lavender",
+            "Never fails to offer comfort to {PRONOUN/m_c/poss} Clanmates",
+            "Is taking {PRONOUN/m_c/poss} sweet time eating {PRONOUN/m_c/poss} meal",
+            "Is purring quietly",
+            "Has been reaching meditative states while sunbathing",
+            "Calmly takes the lead while some Clanmates argue over this and that",
+            "Is unable to be scared, no matter how much the kits try",
+            "Knows everything will be okay in the end",
+            "Takes a deep breath and doesn't let a mean comment get to {PRONOUN/m_c/object}"
+        ],
+        "main_trait_constraint": [
+            "calm"
+        ]
+    },
+    {
+        "id": "careful_gen",
+        "thoughts": [
+            "Is double-checking {PRONOUN/m_c/poss} nest for burrs",
+            "Is getting a small scratch checked out by the medicine cat",
+            "Is slowly and methodically grooming {PRONOUN/m_c/self}",
+            "Is chiding a younger cat for being so reckless",
+            "Is padding back and forth, across the camp",
+            "Is helping to reinforce the nursery walls",
+            "Is helping to reinforce the camp walls",
+            "Is patching up a hole in the camp wall",
+            "Is helping to reinforce the walls of the elders' den",
+            "Fortifies the camp walls for the fourth time this week",
+            "Feels at peace",
+            "Keeps an eye on some rowdy kits"
+        ],
+        "main_trait_constraint": [
+            "careful"
+        ]
+    },
+    {
+        "id": "charismatic_gen",
+        "thoughts": [
+            "Is smiling warmly at {PRONOUN/m_c/poss} Clanmates",
+            "Is making everyone laugh with a very funny comment",
+            "Always manages to turn the mood into a lighter one",
+            "Is laughing with friends",
+            "Is looking around camp for someone to share tongues with",
+            "Winked playfully at a Clanmate from across the clearing!",
+            "Has received several invitations to share tongues... and is deciding which to take",
+            "Is basking in a sunray in the camp clearing",
+            "Has the kits and apprentices very engaged in a very, very tall tale",
```

Comment:
I was thinking an apprentice could be telling a story to other apprentices along the lines of almost catching a huge rabbit or taking down a bear all on their own. I definitely will change the wording/add a new category or two to help make things clearer though

Reply:
update, adding an older charismatic gen section and made it one on one instead of referring to multiple of both as chances are people might not have both